### PR TITLE
#590 - Adapt homepage redesign changes

### PR DESCRIPTION
**Pull request by jonathanbobel**
- [x] Edit the homepage images #521
- [x] Remove account creation option
- [x] Remove navigation
- [x] Remove links to Notify docs in footer

### CODE REVIEW

**Review by ccostino — COMMENTED** [review]
This looks good to me, though I'll have to defer to @stvnrlly on this one for final approval.

I did have one question though, and if it's already been agreed on then no worries!  That said, the `No technical integration needed` section has a screenshot of code next to it and I wonder if that might be confusing to folks.  Would a screenshot or simple image of uploading a CSV be better here?

Also, one small thing we may need to add:  `alt` text for the image; accessibility! :-)

File: app/templates/components/uk_components/footer/template.njk
Comment (stvnrlly) [inline app/templates/components/uk_components/footer/template.njk:66]: Using Jinja comments on plain HTML seems like a good idea, but I also think we should be consistent in comment styling to help our future selves. Let's have this either be an HTML comment or switch the other HTML comments to Jinja comments.

**Review by stvnrlly — APPROVED** [review]
Looks good, just me being annoying about comment consistency

**Comment by jonathanbobel** [comment]
> This looks good to me, though I'll have to defer to @stvnrlly on this one for final approval.
> 
> I did have one question though, and if it's already been agreed on then no worries! That said, the `No technical integration needed` section has a screenshot of code next to it and I wonder if that might be confusing to folks. Would a screenshot or simple image of uploading a CSV be better here?
> 
> Also, one small thing we may need to add: `alt` text for the image; accessibility! :-)

The images all have alt text

**Review by stvnrlly — APPROVED** [review]

**Pull request merged**
